sdk/appcenter-auth/src/main/java/com/microsoft/appcenter/auth/Auth.java

```diff
@@ -479,7 +479,7 @@ private synchronized void initAuthenticationClient(String configurationPayload)
                 throw new IllegalStateException("Cannot find a default b2c or aad authority configured to be the default.");
             }
             mAuthenticationClient = new PublicClientApplication(mContext, getConfigFile());
-            mAuthorityUrl = authorityUrl;
+            mAuthorityUrl = mAuthenticationClient.getConfiguration().getAuthorities().get(0).getAuthorityUri().toString();
```

Comment:
What's the value for AAD there when you test in app?

Reply:
login.microsoftonline.com/{tenantIdGuid}